src/v/pandaproxy/schema_registry/protobuf/.clang-format

```diff
@@ -0,0 +1,2 @@
+Language: Proto
+DisableFormat: true
```

Comment:
could probably add this to the root .clang-format ?

Reply:
Formatting of proto was added in https://github.com/redpanda-data/redpanda/pull/9816, I couldn't find a way to override configuration for subdirectories from the root file.

Reply:
ahh makes sense